src/misc/lv_timer.c

```diff
@@ -267,6 +268,16 @@ void lv_timer_set_repeat_count(lv_timer_t * timer, int32_t repeat_count)
     timer->repeat_count = repeat_count;
 }
 
+/**
```

Comment:
I see that in `lv_timer.c` we heve API comments, so I guess you added it for consistency.
However we should have API comments only in the header files. So could you remove it from here and the other functions in this file as well. :see_no_evil:

Reply:
Yes, I also added it because I saw that it existed in the c file, so I added it hahaha.